docs/src/pages/premium-themes/paperbase/Header.tsx

```diff
@@ -20,7 +20,7 @@ interface HeaderProps {
   onDrawerToggle: () => void;
 }
 
-function Header(props: HeaderProps) {
+export default function Header(props: HeaderProps) {
```

Comment:
The .js file does not appear modified.

Reply:
It's added automatically by the JS generation script. TS is the source of truth.